_site/2015/02/01/13/index.html

```diff
@@ -126,7 +126,7 @@ <h2 id="turtlebot-with-obstacle-avoidance">TurtleBot with Obstacle Avoidance</h2
 
 <div class="highlight"><pre><code class="language-sh" data-lang="sh">roslaunch turtlebot_rviz_launchers view_navigation.launch --screen</code></pre></div>
 
-<p>TurtleBot can’t reliably determine its initial pose (where it is) so <a href="/2015/02/01/12/">we’ll give it a hint</a> by using “2D Pose Estimate”. Select “2D Pose Estimate” and specify TurtleBot’s location. Leave <strong>RViz</strong> open so we can monitor its path planning. </p>
```

Comment:
Actually @markwsilliman it looks like this has the generated outputs committed to the branch. You can let github render it via jekyll and not have to have the html in the repo.